File: backend/Models/HistoryWorkspaceStatus.cs
using System;
using System.Collections.Generic;

namespace backend.Models;

// Представление HistoryWorkspaceStatus из БД
public partial class HistoryWorkspaceStatus
{
    public int? IdWorkspace { get; set; }

    public DateOnly? StartDate { get; set; }

    public DateOnly? EndDate { get; set; }

    public string? WorkerFullName { get; set; }
}

File: frontend/src/app/services/models/HistoryWorkspaceStatus.ts
export interface IHistoryWorkspaceStatus {
    idWorkspace: number;
    idStatusWorkspace?: number;
    startDate?: string; // Используйте строку для даты, так как DateOnly не имеет прямого аналога в JavaScript
    endDate?: string;
    idWorkspaceStatusType?: number;
    workspaceStatusTypeName?: string;
    idWorkspaceReservationsStatuses?: number;
    workspaceReservationStatuseName?: string;
    workerFullName?: string;
    userName?: string;
  }
  